Meta/serenity.sh

```diff
@@ -286,9 +286,26 @@ delete_toolchain() {
 }
 
 kill_tmux_session() {
-    local TMUX_SESSION
-    TMUX_SESSION="$(tmux display-message -p '#S')"
-    [ -z "$TMUX_SESSION" ] || tmux kill-session -t "$TMUX_SESSION"
+    local TRAP_LOCK
+    TRAP_LOCK=/tmp/serenity_tmux_trap.lock
+    [[ ! -f "$TRAP_LOCK" ]] && touch "$TRAP_LOCK"
+
+    exec {lock_fd}<"$TRAP_LOCK"
+    flock -x "$lock_fd"
```

Comment:
Is a lock file actually necessary here? 

We should be able to create a tmux session id and kill the session by id name, rather than asking tmux for the 'current session', no?

`tmux new-session -s serenity-gdb-$(date +%s)` or some such. and then communicate that session name through the subprocesses.

Reply:
True, we don't need flock here, but can we use `$(date +%s)`? There should be different times because we'll launch new instance, so we'll try to kill different sessions